@infotexture   The false positive errors reported in the transformation report have been fixed in dita-ot-gradle v2.8.4.

  What changed

  The error detection logic has been rewritten to use DITA-OT structured message codes only
  ([PREFIX][NUMBER][SEVERITY]). The plugin now parses the severity suffix (I=Info, W=Warning, E=Error, F=Fatal) instead
  of matching the word "error" anywhere in the log output.

  This fixes all 3 reported bugs:
  - Filename false positives — Processing file:/.../error-messages.xml no longer counted as error (no DITA-OT code)
  - Stack trace double-counting — [ERROR] SVG graphic... from Apache FOP and BridgeException stack traces are ignored
  (not DITA-OT codes)
  - "Writing file" false positives — Writing file: messages no longer classified as errors

  Test results (DITA-OT 4.4.0)

  HTML:  0 errors, 2 warnings (was: 6 false errors)
  PDF:   0 errors, 0 warnings (was: 4 false errors)
  
   How to upgrade

  Update build.gradle:

 ```
 plugins {
      id 'io.github.jyjeanne.dita-ot-gradle' version '2.8.4'
  }
```

  Optional — hide warning details in the report:

  ```
task html(type: DitaOtTask) {
      showWarnings false   // shows count only, hides details
  }
```

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
 }
 
 plugins {
-    id 'io.github.jyjeanne.dita-ot-gradle' version '2.8.3'
+    id 'io.github.jyjeanne.dita-ot-gradle' version '2.8.4'
 }
 
 import org.gradle.api.DefaultTask

